create_auction.php

```diff
@@ -28,37 +33,65 @@
         <div class="form-group row">
           <label for="auctionTitle" class="col-sm-2 col-form-label text-right">Title of auction</label>
           <div class="col-sm-10">
-            <input type="text" class="form-control" id="auctionTitle" placeholder="e.g. Black mountain bike">
+            <input type="text" class="form-control" id="auctionTitle" name="auctionTitle" placeholder="e.g. Black mountain bike">
             <small id="titleHelp" class="form-text text-muted"><span class="text-danger">* Required.</span> A short description of the item you're selling, which will display in listings.</small>
           </div>
         </div>
+        <div class="form-group row">
+          <label class="col-sm-2 col-form-label text-right">Select Items</label>

```

Comment:
Since we are restricting users to only select items that have the available status, should we show a small text indicating the same?
Something like "Select Items (Only items not listed in another auctions are visible)"